CHANGELOG.md

```diff
@@ -41,7 +41,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Changed: The connection of EMMI Zone Exit Southeast and EMMI Zone Exit West is now a proper door. This enables it to now be shuffled in door lock rando.
 
 ### Metroid Prime 2: Echoes
+
 - Changed: When Progressive Grapple is enabled, it will now show `2 shuffled copies` rather than `Shuffled` for better consistency.
+- Changed: A proper error message is displayed when mono is not found.
```

Comment:
Maybe mention macos/linux here? since mono is never required on windows, and windows people might be confused on what this means.